libraries/AP_HAL/board/linux.h

```diff
@@ -349,3 +349,7 @@
 
 #define HAL_HAVE_BOARD_VOLTAGE 1
 #define HAL_HAVE_SAFETY_SWITCH 1
+
+#ifndef HAL_BOARD_SHOULD_ZERO_MEMBERS
+#define HAL_BOARD_SHOULD_ZERO_MEMBERS 1
```

Comment:
Because that's how we are going to fix valgrind.... This commit should rather be in the other PR I'm going to create after this one is merged.  I'll remove this commit.